xarray/backends/api.py

```diff
@@ -185,12 +185,28 @@ def _finalize_store(write, store):
     store.close()
 
 
+def load_dataset(*args, **kwargs):
```

Comment:
Maybe this should be: `def load_dataset(filename_or_obj, **kwargs)`? That would be a little more explicit, and ensure that users don't rely on the argument order of keyword arguments to `open_dataset`.

Reply:
Good, idea, changed as suggested